platform/src/main/java/org/hillview/storage/ParquetFileLoader.java

```diff
@@ -127,8 +173,23 @@ private static void appendGroup(
                     break;
                 }
                 case FIXED_LEN_BYTE_ARRAY: {
-                    String s = g.getString(field, 0);
-                    col.append(s);
+                    if (logicalType instanceof IntervalLogicalTypeAnnotation) {
+                        ByteBuffer bb = g.getBinary(field, 0).toByteBuffer();
+                        bb.order(ByteOrder.LITTLE_ENDIAN);
+                        int months = bb.getInt();
+                        int days = bb.getInt();
+                        int milliseconds = bb.getInt();
+                        if (months != 0) {
+                            HillviewLogger.instance.warn("Month is not zero in parquet interval: using conversion of 30 days per month",
```

Comment:
this may trigger millions of warnings when reading a data file. Perhaps log only once.

Reply:
Fixed. Now only log once for each parquet interval column with non-zero month values.